src/accelerator.rs

```diff
@@ -0,0 +1,87 @@
+use drivers::Driver;
+use motions::Motion;
+
+fn clamp(n: isize, min: isize, max: isize) -> isize {
+  if n < min {
+    return min;
+  }
+  if n > max {
+    return max;
+  }
+  n
```

Comment:
n is the input, in rust you dont have to `return n;` you can just leave it at `n` and it means the same